src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -144,16 +145,19 @@ class ActiveFocusInvestigation extends React.Component<
     ]);
     supersetService(SUPERSET_PLANS_SLICE, supersetParams)
       .then((result: Plan[]) => {
```

Comment:
@kelvin-muchiri should we also change this type. what do we expect to recieve when there is an error?

Reply:
When the result is falsy it returns false

Reply:
When 401 error is encountered the result is **undefined**

Reply:
@kelvin-muchiri then the the type should be `Plan[] | undefined`, yes? because otherwise `Plan[]` is a type that is forever truthy.